omnigibson/robots/tiago.py

```diff
@@ -282,7 +280,7 @@ def _post_load(self):
             self.eef_links[arm].visual_only = True
             self.eef_links[arm].visible = False
 
-        self._world_base_fixed_joint_prim = get_prim_at_path(os.path.join(self.root_link.prim_path, "world_base_joint"))
+        self._world_base_fixed_joint_prim = get_prim_at_path(f"{self._prim_path}/rootJoint")
```

Comment:
Did you also remove the fixed joint from Tiago in the `omni` branch of `ig_assets`?